packages/host/app/components/operator-mode/detail-panel.gts

```diff
@@ -88,89 +95,198 @@ export default class DetailPanel extends Component<Signature> {
-          <h3>Inheritance Panel</h3>
-          <ModuleDefinitionContainer
-            @fileURL={{this.cardType.type.module}}
-            @name={{this.cardType.type.displayName}}
-            @fileExtension={{this.cardType.type.moduleMeta.extension}}
-            @infoText={{this.lastModified.value}}
-            @isActive={{true}}
-            @actions={{array
-              (hash label='Delete' handler=@delete icon='icon-trash')
-            }}
-          />
-          {{#if this.cardType.type.super}}
-            <div>Inherits from</div>
-            <ClickableModuleDefinitionContainer
-              @fileURL={{this.cardType.type.super.module}}
-              @name={{this.cardType.type.super.displayName}}
-              @fileExtension={{this.cardType.type.super.moduleMeta.extension}}
-              @onSelectDefinition={{this.updateCodePath}}
-              @url={{this.cardType.type.super.module}}
-            />
-          {{/if}}
-        {{else if this.isBinary}}
-          <FileDefinitionContainer
-            @fileURL={{@readyFile.url}}
-            @fileExtension={{this.fileExtension}}
-            @infoText={{this.lastModified.value}}
-            @actions={{array
-              (hash label='Delete' handler=@delete icon='icon-trash')
-            }}
-          />
         {{/if}}
       {{/if}}
     </div>
     <style>
-      .container {
+      .header {
+        --boxel-header-text-size: var(--boxel-font-size-sm);
+        --boxel-header-padding: var(--boxel-sp-xs);
+        --boxel-header-text-size: var(--boxel-font-size-sm);
+        --boxel-header-text-transform: uppercase;
+        --boxel-header-letter-spacing: var(--boxel-lsp-xxl);
+        --boxel-header-background-color: var(--boxel-100);
+        --boxel-header-text-color: var(--boxel-dark);
+        --boxel-header-max-width: none;
+      }
+      .in-this-file-panel-banner {
         display: flex;
-        flex-direction: column;
-        gap: var(--boxel-sp-xs);
+        justify-content: space-between;
+        align-items: center;
+      }
+      .panel-header {
+        font: 700 var(--boxel-font);
+        letter-spacing: var(--boxel-lsp-xs);
+      }
+      .number-items {
+        color: #919191;
+        font-size: var(--boxel-font-size-sm);
+        font-weight: 200;
+        letter-spacing: var(--boxel-lsp-xxl);
+        text-transform: uppercase;
       }
       .selected {
         outline: 2px solid var(--boxel-highlight);
       }
-      .inheritance-chain {
-        padding: var(--boxel-sp-sm);
+      .in-this-file-panel,
+      .details-panel,
+      .inheritance-panel {
+        padding-top: var(--boxel-sp-sm);
+        gap: var(--boxel-sp-xs);
+        display: flex;
+        flex-direction: column;
+      }
+      .in-this-file-menu {
+        padding: var(--boxel-sp-xs);
+      }
+      .loading {
+        display: flex;
+        justify-content: center;
+      }
+      .chain {
+        display: flex;
+        font-size: var(--boxel-font-size);
```

Comment:
Isn't this supposed to be smaller, about 13px? I'd use `var(--boxel-font-size-sm)`. If you want the corresponding line-height to be set as well, you can use `font` shorthand instead of `font-size`. In that case the css would be `font: var(--boxel-font-sm);`